__DOCS__/examples/annotatedExamples/configuration.md

```diff
@@ -19,16 +19,5 @@ const cld = new Cloudinary({
 <h2>2. Instance configuration</h2>
 
 ```javascript
-
-// Import the asset you want to use
-// In this case, we import a CloudinaryImage type.
-import {CloudinaryImage} from "@cloudinary/base";
-
-const image = new CloudinaryImage('sample', {
-    // any cloudConfiguration goes here
```

Comment:
Maybe remove this whole section, no?
since this refers to instance configuration

Reply:
deleted